### ISSUE TITLE
Installer error message when Tk isn't installed

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6855]
|<img alt="mboersma's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| mboersma reported|
|-|-|
|Trac ID|trac#282|
|Type|defect|
|Created|2003-09-24 23:20:01Z|

Attachments:

* [282.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/267/267b976f6f335984ab90f0f478e8a1637eabe7d5/380aca700f910b150b7f4751a41b0b4c24bed450.diff) (5048 bytes) - added by moonfallen on 2003-09-28 11:53:20Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__282 282
type__defect defect
reporter__mboersma mboersma
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1064445601000000 1064445601000000
changetime__1064888620000000 1064888620000000
version__ 
owner__moonfallen moonfallen
cc__moonfallen cc__mboersma
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="mboersma's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|mboersma commented|
|-|-|

```
#!html
<pre>
On Windows, the python 2.3 binary installer allows you *not*
to install Tkinter support by clicking a checkbox.  (And
thanks to wxPython, that's a box we like to click.)
If python is installed this way, the Twisted 1.0.7 binary
installer will cause an error looking for tk84.dll.  It
seems to be a benign problem.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
Fixed.  attached patch for monday's build.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
Testing on another system I discovered that I actually still
get the popup error even though the fallback works. 
Changing to cbb because I want to get rid of the error
message.  This probably means searching the binary path for
the missing library before importing Tkinter.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
Completely fixed; alpha3 (not alpha2) uploaded to
sourceforget.  (You will see these rsn; alpha3 hasn't been
posted just yet.)

Incidentally, I had to implement a horrible kludge because
of this:
https://sourceforge.net/tracker/index.php?func=detail&amp;aid=814654&amp;group_id=5470&amp;atid=105470

With luck my bug report will get resolved in 2.3.2 or so and
I can take out the kludge.

</pre>
```
